main.js

```diff
@@ -13,6 +14,11 @@ console.log('channels: ' + JSON.stringify(JSON.parse(process.env.TWITCH_CHANNELS
 let loggedIn = false;
 let browser;
 let page;
+let date = new Date();
+
+var prevCoins = 0;
```

Comment:
You should use ```let``` instead of ```var```. ```var``` makes the variable global which is the same as using ```let``` in the highest scope so it's considered bad practice